packages/core/src/sass/app/_common/components/onfido-container.scss

```diff
@@ -194,19 +194,24 @@
 
         @include desktop() {
             grid-template-areas: 'section-side-note section-content';
-            grid-template-columns: auto 1fr;
-            grid-gap: 1rem;
         }
 
-        align-items: center;
-
         @include mobile {
             grid-template-areas: 'section-side-note' 'section-content';
         }
 
         &-side-note {
             grid-area: section-side-note;
+            width: 26rem;
+
+            .dc-text {
```

Comment:
if it is related to 'Text' component, can we use there the prop 'size' ?

Reply:
same with the color prop

Reply:
@shahzaib-deriv, something like this `size={isMobile() ? 'xxs' : 'xs'}`